Assets/Packages/com.studio23.ss2.authsystem.steam/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "com.studio23.ss2.authsystem.steam",
-  "version": "0.1",
+  "version": "0.9.0",
   "displayName": "AuthenticationSteam",
```

Comment:
Change displayName. 

Possible display name for this asset, `"displayName": "Authentication System - Steam"`